test/plugins/individual-collateral/collateralTests.ts

```diff
@@ -590,5 +609,361 @@ export default function fn<X extends CollateralFixtureContext>(
+        },
+        minTradeVolume: bn('0'), // $0
+        rTokenMaxTradeVolume: MAX_UINT192, // +inf
+        shortFreeze: bn('259200'), // 3 days
+        longFreeze: bn('2592000'), // 30 days
+        rewardRatio: bn('1069671574938'), // approx. half life of 90 days
+        unstakingDelay: bn('1209600'), // 2 weeks
+        withdrawalLeak: fp('0'), // 0%; always refresh
+        warmupPeriod: bn('60'), // (the delay _after_ SOUND was regained)
+        tradingDelay: bn('0'), // (the delay _after_ default has been confirmed)
+        batchAuctionLength: bn('900'), // 15 minutes
+        dutchAuctionLength: bn('1800'), // 30 minutes
+        backingBuffer: fp('0'), // 0%
+        maxTradeSlippage: fp('0.01'), // 1%
+        issuanceThrottle: {
+          amtRate: fp('1e6'), // 1M RToken
+          pctRate: fp('0.05'), // 5%
+        },
+        redemptionThrottle: {
+          amtRate: fp('1e6'), // 1M RToken
+          pctRate: fp('0.05'), // 5%
+        },
+      }
+
+      interface IntegrationFixture {
+        ctx: X
+        protocol: DefaultFixture
+      }
+
+      const integrationFixture: Fixture<IntegrationFixture> =
+        async function (): Promise<IntegrationFixture> {
+          return {
+            ctx: await loadFixture(
+              makeCollateralFixtureContext(owner, { maxTradeVolume: MAX_UINT192 })
+            ),
+            protocol: await loadFixture(defaultFixture),
+          }
+        }
+
+      before(async () => {
+        defaultFixture = await getDefaultFixture(collateralName)
+        chainId = await getChainId(hre)
+        if (useEnv('FORK_NETWORK').toLowerCase() === 'base') chainId = 8453
+        if (!networkConfig[chainId]) {
+          throw new Error(`Missing network configuration for ${hre.network.name}`)
+        }
+        ;[, owner, addr1] = await ethers.getSigners()
+      })
+
+      beforeEach(async () => {
+        let protocol: DefaultFixture
+        ;({ ctx, protocol } = await loadFixture(integrationFixture))
+        ;({ collateral } = ctx)
+        ;({ deployer, facadeWrite, govParams } = protocol)
+
+        supply = fp('1')
+
+        // Create a paired collateral of the same targetName
+        pairedColl = await makePairedCollateral(await collateral.targetName())
+        await pairedColl.refresh()
+        expect(await pairedColl.status()).to.equal(CollateralStatus.SOUND)
+        pairedERC20 = await ethers.getContractAt('ERC20Mock', await pairedColl.erc20())
+
+        // Prep collateral
+        collateralERC20 = await ethers.getContractAt('IERC20Metadata', await collateral.erc20())
+        await mintCollateralTo(
+          ctx,
+          toBNDecimals(fp('1'), await collateralERC20.decimals()),
+          addr1,
+          addr1.address
+        )
+
+        // Set primary basket
+        const rTokenSetup: IRTokenSetup = {
+          assets: [],
+          primaryBasket: [collateral.address, pairedColl.address],
+          weights: [fp('0.5e-4'), fp('0.5e-4')],
```

Comment:
why e-4?

Reply:
There was a delicate balancing act trying to find the right RToken supply + basket weights that would work with all token decimals plus max holder balances from mainnet. This is what I found worked for all tokens. It took lots of trial and error.